HdmiCecSink/HdmiCecSink.cpp

```diff
@@ -461,11 +461,20 @@ namespace WPEFramework
        void HdmiCecSinkProcessor::process (const InitiateArc &msg, const Header &header)
        {
             printHeader(header);
+            PhysicalAddress physical_addr_invalid = {0x0F,0x0F,0x0F,0x0F};
+            PhysicalAddress physical_addr_arc_port = {0x02,0x00,0x00,0x00};
```

Comment:
Why Hardcoded physical address?

Reply:
Hi amit,   I have defined a high level task ticket to remove hardcoding of HDMI ARC/eARC port from Displaysettings, HdmiCecSink & Devicesettings.  It's tracked under : https://ccp.sys.comcast.net/browse/RDKTV-16277